### ISSUE TITLE
[Guidelines] 5b5+++ and 5b5++++ consistency

### ISSUE THREAD

**Issue by pedrosino** [thewca/wca-regulations#370]
5b5+ and 5b5++ start exactly the same way (Examples of parts physically detached, resulting in a", but this is not the case for 5b5+++ and 5b5++++

```
5b5+++) EXAMPLE Example of parts not fully attached, resulting in a solved state: a 5x5x5 centre piece twisted in its spot.
5b5++++) EXAMPLE Example of parts physically attached but not fully placed, resulting in a "DNF": a 3x3x3 edge piece slightly popped out.
```

In my understanding, they should start the same way, only with the result being different.

**Comment by viroulep** [comment]
If I get this correctly, you suggest rewording 5b5+++ to:

```
5b5+++) EXAMPLE Example of parts physically attached but not fully placed, resulting in a solved state: a 5x5x5 centre piece twisted in its spot.
```

The point of 5b5+++ and 5b5++++ being to clarify what is a "not fully placed" piece, that makes sense.

10d uses "fully attached"; I think it both "fully attached" and "fully placed" refer to the same idea, so we should probably also have only the distinction between "physically attached" and "fully placed", and get rid of the use of "fully attached".

**Comment by pedrosino** [comment]
That is right.
